apps/portals/src/react-app-env.d.ts

```diff
@@ -1,7 +1,17 @@
-/// <reference types="react-scripts" />
-declare var pluralize: any
+declare let pluralize: any
 
 declare module 'plotly.js-basic-dist' {
   import PlotlyTyped from 'plotly.js'
   export default PlotlyTyped
 }
+
+/// <reference types="vite/client" />
+interface ImportMetaEnv {
+  readonly VITE_PORTAL_NAME: string
+  readonly VITE_PORTAL_DESCRIPTION: string
+  readonly VITE_PORTAL_KEY: string
+}
+
+interface ImportMeta {
+  readonly env: ImportMetaEnv
+}
```

Comment:
These declarations allow us to access the environment variables in a type-safe way.